arviz/plots/pairplot.py

```diff
@@ -56,15 +57,15 @@ def plot_pair(
         If None, size is (8 + numvars, 8 + numvars)
     textsize: int
         Text size for labels. If None it will be autoscaled based on figsize.
-    kind : str
+    kind : str or List[str]
         Type of plot to display (scatter, kde and/or hexbin)
     gridsize : int or (int, int), optional
         Only works for kind=hexbin.
         The number of hexagons in the x-direction. The corresponding number of hexagons in the
         y-direction is chosen such that the hexagons are approximately regular.
         Alternatively, gridsize can be a tuple with two elements specifying the number of hexagons
         in the x-direction and the y-direction.
-    contour : bool
+    contour : bool, optional, deprecated, Defaults to True.
```

Comment:
You wrote it defaults to `True` here but set the default to `None` in the function definition above

Reply:
It actually _does_  default to `True`. But if I let the argument parser handle defaulting, we get the bogus warning. So I have to turn `None` into `True` myself, in code.

Reply:
Yeah I saw that in your code -- just saying that, as is, the docstring contradicts the type hints for the function. Not a big deal I think though, this shouldn't delay merging.

Reply:
I just pushed a quick clarification in the docstring. It was easy to make, and it will avoid confusion.  Thanks.